src/mudlet.pro

```diff
@@ -119,39 +119,25 @@ BUILD_COMMIT_TEST = $$lower($$BUILD_COMMIT_TEST)
 }
 
 
-########################## Version and Build setting ###########################
```

Comment:
There are other "section" dividers like this that I put into this file - I'd prefer to keep them as it helps when scrolling through the file fast (my mouse is set to unlock it's vertical wheel racket so it spins freely when spun hard 😜).

Reply:
Apologies, but if we need to have dividers then it is a sign the file is too complicated and too long - and needs to be simplified.

Reply:
So you are going to remove them *all* (and the explanation texts) and hope that everyone who needs to work out what to do can do so *without* them in the future? Where to strike a balance...

From: https://xkcd.com/2730/:
![code_lifespan-1](https://github.com/Mudlet/Mudlet/assets/6163092/7cd86036-223e-47d5-9803-ecc92978544d)

Reply:
I don't envision putting in that much more work into qmake that we depreciated four years ago and that KDE has moved off as well. At some point we do need to learn how cmake works...